types/src/vm/modules.rs

```diff
@@ -108,6 +100,11 @@ impl ModuleStorageEntry {
         &self.serialized_module
     }
 
+    /// Returns the hash of the given module.
+    pub fn hash(&self) -> [u8; 32] {
```

Comment:
question: could it make sense to return &self.hash / &[u8; 32]? (can be copied on caller side if needed?)

Reply:
done